composer/datasets/c4_hparams.py

```diff
@@ -72,16 +74,31 @@ def initialize_object(self, batch_size: int, dataloader_hparams: DataLoaderHpara
             raise MissingConditionalImportError(extra_deps_group='nlp', conda_package='transformers') from e
 
         # Get StreamingC4 dataset
-        dataset = StreamingC4(remote=self.remote,
-                              local=self.local,
-                              split=self.split,
-                              shuffle=self.shuffle,
-                              tokenizer_name=self.tokenizer_name,
-                              max_seq_len=self.max_seq_len,
-                              group_method=self.group_method,
-                              max_retries=self.max_retries,
-                              timeout=self.timeout,
-                              batch_size=batch_size)
+        if self.version == 1:
+            dataset = StreamingC4(remote=self.remote,
+                                  local=self.local,
+                                  split=self.split,
+                                  shuffle=self.shuffle,
+                                  tokenizer_name=self.tokenizer_name,
+                                  max_seq_len=self.max_seq_len,
+                                  group_method=self.group_method,
+                                  max_retries=self.max_retries,
+                                  timeout=self.timeout,
+                                  batch_size=batch_size)
+        elif self.version == 2:
+            from streaming.text import C4
```

Comment:
do we have a pip package set up for streaming yet? This should be:
```
try:
    import <package name>
except ImportError:
    raise ...
```

Reply:
We have the setup.py but not the PyPI name

Reply:
Done